config/settings/development.yml

```diff
@@ -1,8 +1,13 @@
-moab:
-  # this is a list of named storage roots.  the names are used to seed the endpoints table.
-  storage_roots:
-    # name: '/location/on/disk'  # e.g.
+storage_root_map:
+  default:
     fixture_sr1: 'spec/fixtures/storage_root01'
     fixture_sr2: 'spec/fixtures/storage_root02'
     fixture_sr3: 'spec/fixtures/checksum_root01'
+  preservation_catalog_prod:
+    fixture_sr1: 'spec/fixtures/storage_root01'
+    fixture_sr2: 'spec/fixtures/storage_root02'
+    fixture_sr3: 'spec/fixtures/checksum_root01'
+  preservation_catalog_prod_a:
+    fixture_sr1: 'spec/fixtures/storage_root01'
+    fixture_sr2: 'spec/fixtures/storage_root02'
```

Comment:
Are we not adding the checksum_root01 b/c the objects in there arent valid?

Reply:
The blocks for `preservation_catalog_prod` and `prod_a` here are just to show an example.  They could just as easily be like

```yaml
preservation_catalog_prod_a:
    fixture_sr1: 'first/storage/root/path'
    fixture_sr2: 'second/storage/root/path'
    fixture_sr3: 'third/storage/root/path'
```
Would it help to write them up like that, and maybe include a comment?

Reply:
Ohhhh gotcha that makes sense. Yeah sure either or works for me :)